src/domains/plugin/components/PluginImage/PluginImage.tsx

```diff
@@ -26,7 +31,7 @@ export const PluginImage = ({
 	logoURL,
 	isUpdating,
 	updatingProgress,
```

Comment:
```suggestion
	updatingProgress = 0,
```
good to use the default value in getting props instead of `||` everywhere in component